source/reflect.h2

```diff
@@ -425,6 +425,16 @@ add_virtual_destructor: (inout t: meta::type_declaration) =
                "could not add virtual destructor");
 }
 
+details: namespace =
+{
+
+add_default_constructor: (inout t: meta::type_declaration) =
```

Comment:
This is in the right place, but I wouldn't use a `details` subnamespace... I think it's just like the `add_virtual_destructor` above it, a free-function helper that would be useful for other metafunction writers.

Reply:
Excellent. Turns out there's already an use for it in `basic_value`.